java-frontend/src/test/java/org/sonar/java/resolve/SymbolTableTest.java

```diff
@@ -1146,6 +1147,52 @@ public void infer_fully_lambda_types() {
     JavaType Rsubstitution = typeSubstitution.substitutedTypes().get(1);
```

Comment:
if it wasn't in a test yes it would, but here, as the type var is named R I made an exception.